check/format-incremental

```diff
@@ -87,14 +87,23 @@ if (( only_changed == 1 )); then
 else
```

Comment:
The if-branch above (lines 84-86) does not include `.ipynb` files.  Please correct.